graylog2-web-interface/src/views/pages/ExtendedSearchPage.jsx

```diff
@@ -112,14 +122,30 @@ const DashboardSearchBarWithStatus = WithSearchStatus(DashboardSearchBar);
 
 const ViewAdditionalContextProvider = connect(AdditionalContext.Provider, { view: ViewStore }, ({ view }) => ({ value: { view: view.view } }));
 
-const ExtendedSearchPage = ({ route, searchRefreshHooks }: Props) => {
+const useStyle = () => {
+  useEffect(() => {
+    style.use();
+    return () => style.unuse();
+  }, []);
+};
+
+const ExtendedSearchPage = ({ route, location = { query: {} }, router, searchRefreshHooks }: Props) => {
+  const { pathname, search } = router.getCurrentLocation();
+  const query = `${pathname}${search}`;
   const refreshIfNotUndeclared = () => _refreshIfNotUndeclared(searchRefreshHooks, SearchExecutionStateStore.getInitialState());
 
   useEffect(() => {
-    style.use();
+    const { view } = ViewStore.getInitialState();
 
+    bindSearchParamsFromQuery({ view, query: location.query, retry: () => Promise.resolve() });
+  }, [query]);
```

Comment:
One thing I realised while testing these changes:

Changing e.g. the relative time range, results in a new URL due to `useSyncWithQueryParameters`. because of the new URL, the query variable is different and `bindSearchParamsFromQuery ` gets called again.

`bindSearchParamsFromQuery` will unnecessarily update the current query and also trigger another, but unnecessary search execution.